test/testdata/resolver/bad_param_ordering.rb

```diff
@@ -32,7 +32,7 @@ def f2(b, a); end
 
   sig {params(v: T1, w: T2, x: T1, y: T2, z: T3).void}
   def f3(v, w = T2.new, *y, z:, x: T1.new); end
-  #                      ^ error: Bad parameter ordering for `y`, expected `x` instead
-  #                         ^^ error: Bad parameter ordering for `z`, expected `y` instead
-  #                             ^^ error: Bad parameter ordering for `x`, expected `z` instead
+  #                      ^ error-with-dupes: Bad parameter ordering for `y`, expected `x` instead
+  #                         ^^ error-with-dupes: Bad parameter ordering for `z`, expected `y` instead
+  #                             ^^ error-with-dupes: Bad parameter ordering for `x`, expected `z` instead
```

Comment:
Are these repeats introduced because there are multiple methods with bad ordering now?

Reply:
yes, the method is duplicated, once for each default now